rskj-core/src/test/java/co/rsk/peg/bitcoin/ErpRedeemScriptTestUtils.java

```diff
@@ -0,0 +1,59 @@
+package co.rsk.peg.bitcoin;
+
+import co.rsk.bitcoinj.core.BtcECKey;
+import co.rsk.bitcoinj.core.Utils;
+import co.rsk.bitcoinj.script.ScriptOpCodes;
+
+import java.math.BigInteger;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ErpRedeemScriptTestUtils {
+    public static void assertEmergencyRedeemScript(byte[] redeemScript, List<BtcECKey> emergencyPublicKeys, int lastScriptIndex, int erpThreshold) {
```

Comment:
```suggestion
    public static void assertStandardMultiSigRedeemScript(byte[] redeemScript, List<BtcECKey> emergencyPublicKeys, int lastScriptIndex, int erpThreshold) {
```

you are not checking anything erp-related right? just the normal structure of a M/N multisig.

Reply:
done